modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1344,6 +1344,10 @@ void ONNXImporter::populateNet(Net dstNet)
             if (node_proto.input_size() == 2) {
                 Mat blob = getBlob(node_proto, constBlobs, 1);
                 CV_Assert(blob.type() == CV_32SC1);
+                if (std::find(blob.ptr<int>(), blob.ptr<int>() + blob.total(), -1) == blob.ptr<int>() + blob.total())
+                {
+                    std::replace(blob.ptr<int>(), blob.ptr<int>() + blob.total(), 0, -1);
+                }
```

Comment:
May I ask to comment this condition?

Reply:
I'll add comment.
I can replace condition:
`if (std::find(blob.ptr<int>(), blob.ptr<int>() + blob.total(), -1) == blob.ptr<int>() + blob.total())`
to 
`if (framework_name == "keras2onnx")`
Which of the conditions is better?

Reply:
It's always better to keep generic condition. But it depends on the issue we try to resolve (Keras2ONNX bug or ONNX import)